pkg/log/log_test.go

```diff
@@ -6,12 +6,24 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"math"
+	"testing"
 
 	"github.com/getoutreach/gobox/pkg/app"
 	"github.com/getoutreach/gobox/pkg/log"
 	"github.com/getoutreach/gobox/pkg/log/logtest"
 )
 
+func TestUnmarshalableValues(t *testing.T) {
+	t.Run("Infinity", func(t *testing.T) {
+		log.Info(context.Background(), "infinity is not fine but not a problem", log.F{"party": math.Inf(1)})
```

Comment:
Writing to `b` should allow us to test that the value was written in an expected format as well.

Reply:
done.